src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.cpp

```diff
@@ -126,10 +126,10 @@ char calculateInitialMissedHbCounter(const mqbcfg::TcpInterfaceConfig& config)
     // we calculate initial value of 'MissedHbCount' like so (taking into
     // account the possibility of overflow):
 
-    const char retVal = bsl::min(
-        bsl::numeric_limits<char>::max(),
-        static_cast<char>((3 * bdlt::TimeUnitRatio::k_MS_PER_M) /
-                          config.heartbeatIntervalMs()));
+    const int retVal = bsl::min(
+        bsl::numeric_limits<int>::max(),
+        static_cast<int>((3 * bdlt::TimeUnitRatio::k_MS_PER_M) /
+                         config.heartbeatIntervalMs()));
```

Comment:
Note that we have 3 minutes here, and just 10 seconds for client connections.
Is it intended?